static/app/views/metrics/pageHeaderActions.tsx

```diff
@@ -96,7 +96,7 @@ export function PageHeaderActions({showAddMetricButton, addCustomMetric}: Props)
       {
         leadingItems: [<IconSettings key="icon" />],
         key: 'configure-metric',
-        label: t('Configure Metric'),
+        label: t('Show All Metrics'),
```

Comment:
Just had a talk with Ale about this, it should be:
```suggestion
        label: t('Metrics Settings'),
```